mmv1/third_party/terraform/website/docs/r/dataproc_workflow_template.html.markdown

```diff
@@ -782,143 +782,147 @@ cluster_config {
-    
+  The Cloud Storage URI of the keystore file used for SSL encryption. If not provided, Dataproc will provide a self-signed certificate.
+
 * `keystore_password` -
   (Optional)
-  Optional. The Cloud Storage URI of a KMS encrypted file containing the password to the user provided keystore. For the self-signed certificate, this password is generated by Dataproc.
-    
+  The Cloud Storage URI of a KMS encrypted file containing the password to the user provided keystore. For the self-signed certificate, this password is generated by Dataproc.
+
 * `kms_key` -
   (Optional)
-  Optional. The uri of the KMS key used to encrypt various sensitive files.
-    
+  The uri of the KMS key used to encrypt various sensitive files.
+
 * `realm` -
   (Optional)
-  Optional. The name of the on-cluster Kerberos realm. If not specified, the uppercased domain of hostnames will be the realm.
-    
+  The name of the on-cluster Kerberos realm. If not specified, the uppercased domain of hostnames will be the realm.
+
 * `root_principal_password` -
   (Optional)
-  Optional. The Cloud Storage URI of a KMS encrypted file containing the root principal password.
-    
+  The Cloud Storage URI of a KMS encrypted file containing the root principal password.
+
 * `tgt_lifetime_hours` -
   (Optional)
-  Optional. The lifetime of the ticket granting ticket, in hours. If not specified, or user specifies 0, then default value 10 will be used.
-    
+  The lifetime of the ticket granting ticket, in hours. If not specified, or user specifies 0, then default value 10 will be used.
+
 * `truststore` -
   (Optional)
-  Optional. The Cloud Storage URI of the truststore file used for SSL encryption. If not provided, Dataproc will provide a self-signed certificate.
-    
+  The Cloud Storage URI of the truststore file used for SSL encryption. If not provided, Dataproc will provide a self-signed certificate.
+
 * `truststore_password` -
   (Optional)
-  Optional. The Cloud Storage URI of a KMS encrypted file containing the password to the user provided truststore. For the self-signed certificate, this password is generated by Dataproc.
-    
+  The Cloud Storage URI of a KMS encrypted file containing the password to the user provided truststore. For the self-signed certificate, this password is generated by Dataproc.
+
 The `software_config` block supports:
-    
+
 * `image_version` -
   (Optional)
-  Optional. The version of software inside the cluster. It must be one of the supported (https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions#other_versions). If unspecified, it defaults to the latest Debian version.
-    
+  The version of software inside the cluster. It must be one of the supported (https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions#other_versions). If unspecified, it defaults to the latest Debian version.
```

Comment:
Might as well update this to match https://cloud.google.com/dataproc/docs/reference/rest/v1/ClusterConfig#SoftwareConfig

```suggestion
  The version of software inside the cluster. It must be one of the supported [Dataproc Versions](https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions#supported_dataproc_versions), such as "1.2" (including a subminor version, such as "1.2.29"), or the ["preview" version](https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions#other_versions). If unspecified, it defaults to the latest Debian version.
```